game/CMakeLists.txt

```diff
@@ -93,6 +93,12 @@ foreach(lib ${OUR_LIBS} SDL2 PangoCairo LibRSVG LibXML++ AVFormat SWResample SWS
 	add_definitions(${${lib}_DEFINITIONS})
 endforeach(lib)
 
+if(LibXML++_VERSION_2_6)
+  set(LIBXMLPP_VERSION 2)
+else()
+  set(LIBXMLPP_VERSION 3)
```

Comment:
This isn't really necessary but I guess it makes it more readable?

Reply:
This review should be in #492, not here. I don't recall if there was a reason for this change other than cleanup.

Reply:
Fair enough. Anyway, it's just a comment.